README.md

```diff
@@ -129,102 +121,126 @@ service:
+
+For instance, if you'd like to add a TypeScript generator to your project, simply run:
 
-```bash
-fern generate
+```
+fern add fern-typescript-sdk
 ```
 
-And voila! You just built and published a TypeScript SDK.
+Which will add the Node variant of the TypeScript SDK to the list of generators - the other variant being a TypeScript browser library, to be used by frontends directly.
 
-**Next step:** define _your_ API in Fern. Check out our [docs](https://www.buildwithfern.com/docs/definition) to learn more.
+Your `generators.yml` file should contain the new generator:
 
-# CLI reference
+```
+default-group: local
+groups:
+  local:
+    generators:
+      - name: fernapi/fern-typescript-node-sdk
+        version: 0.6.4
+        output:
+          location: local-file-system
+          path: ../../generated/typescript
+```
 
-### `fern generate [--group <group>] [--version <version>]`
+Note the `local-file-system` output may not be present by default. If it's not, simply add it and specify a local path to where the generated client will be saved.
 
-`fern generate` runs the compiler. It will validate your API and run the
-generators specified in `generators.yml`.
+## Publishing Clients
 
-If you don't have a `default-group` specified in `generators.yml`, then you must
-specify a group using the `--group` option.
+Fern supports automatically publishing clients to an NPM registry. By default, Node packages are published to `npm.buildwithfern.com` and are named `@organisation-fern/package`
 
-You can specify a version using the `--version` option. This version string is
-used when publishing SDKs to registries (e.g. npm).
+That's all there's to it.
 
-#### Running locally
+## Next Steps
 
-By default, the generation runs in the cloud. If you want to run it on your local machine, you can use the `--local` option. This will run each generator in a Docker container.
+Define _your_ API in Fern. Check out our [﻿Definition Documentation](https://www.buildwithfern.com/docs/definition) to learn more.
 
-By default, Fern will delete the container after running. To keep the container around (e.g. to look at the generator's logs), use the `--keepDocker` option.
+# Examples
 
----
+Our Plant Store provides a full example of both an API defined in Fern, as well as the SDKs generated from it in both TypeScript and Java. We have also included the generated Postman Collections and OpenAPI Specification that was generated from the Fern spec.
 
-### `fern check`
+-   [﻿Generated TypeScript SDK](https://github.com/fern-api/plantstore-node)    
+-   [﻿Generated Java SDK](https://github.com/fern-api/plantstore-java)    
+-   [﻿Generated Postman Collection](https://github.com/fern-api/plantstore-postman)    
+-   [﻿Generated OpenAPI spec](https://github.com/fern-api/plantstore-openapi)
```

Comment:
@aevitas i could see ditching the Plantstore example and showing real examples of folks using Fern. 

Dropping some links for you here: 
- Flatfile TypeScript SDK (https://github.com/FlatFilers/flatfile-node)
- Vellum Python SDK (https://pypi.org/project/vellum-ai/) 
- Vellum Docs (https://docs.vellum.ai/api-reference/generate)
- Flipt Java SDK (https://github.com/flipt-io/flipt-java)

Reply:
For sure - I've replaced the plant store with the real world examples.